### DISCUSSION TITLE
User Created and Maintained Markdown Pages

### DISCUSSION THREAD

**Discussion by BelieveInBunny** [torrust/torrust-index-gui#89] [Feature requests]
Allowing users to create pages using markdown

**Comment by ldpr** [comment]
@WarmBeer do you happen to know the status on this? We currently run a subdomain solely for docs & troubleshooting etc, we'd love to slot it into torrust instead in a few custom created pages.

**Comment by josecelano** [comment]
> @WarmBeer do you happen to know the status on this? We currently run a subdomain solely for docs & troubleshooting etc, we'd love to slot it into torrust instead in a few custom created pages.

Hi, @ldpr right now there is only support for:

- Markdown for the torrent description.
- Including proxied images in torrent description

**Comment by josecelano** [comment]
Hi @BelieveInBunny @ldpr, what type of pages can users create? I mean, is not this a duplicate feature of https://github.com/torrust/torrust-index-gui/discussions/291

Did you mean "admin" users?

I'm closing the issue assuming that only admins can create these pages.

Feel free to re-open if it's a different case.

**Reply by IAmBlackat** [reply]
@josecelano yes sir, I mean by the admin users so they can add pages about them, service or what the index is about.

**Reply by josecelano** [reply]
Unfortunately, it's out-of-scope in the short term. See https://github.com/torrust/torrust-index-gui/issues/657. But definitively a nice-to-have.
